assets/ACS2/acs2++.cc

```diff
@@ -311,14 +311,11 @@ int startActionPlanning(ClassifierList *population, Environment *env, int time,
         } else {
             Action **actSequence = population->searchGoalSequence(situation, goalSituation);
             int i;
-            //cout<<situation<<"->";
```

Comment:
Mimo że jest zakomentowane to raczej bym to zostawił. To oryginalny kod autora - może ktoś kiedyś się zainspiruje tym fragmentem.